db/seeds.rb

```diff
@@ -110,10 +110,10 @@
 trip13.image = image13
 trip13.save
 
-image14 = File.open('app/assets/images/positano.jpg')
+image14 = File.open('app/assets/images/positano-it.jpg')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.